CMakeLists.txt

```diff
@@ -312,7 +312,15 @@ endif()
 add_subdirectory(libraries/rainbow) # Qt extension for colors
 add_subdirectory(libraries/LocalPeer) # fork of a library from Qt solutions
 add_subdirectory(libraries/classparser) # class parser library
-add_subdirectory(libraries/tomlc99) # toml parser
+if(NOT Launcher_FORCE_BUNDLED_LIBS)
+  find_package(tomlplusplus 3.2.0 QUIET)
+endif()
```

Comment:
I would prefer this to be further up. Maybe right after the huge block for finding Qt (so at line 193)